packages/flame_spine/example/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Add a note above this that it applies to the spine animation and then on the top of this document you add our normal licence